modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -1388,7 +1400,7 @@ private ObjectEntry _expireObjectEntry(
 		_checkObjectEntryObjectDefinitionId(
 			objectDefinition, serviceBuilderObjectEntry);
 
-		_objectEntryService.expireObjectEntry(
+		_objectEntryService.expireObjectEntryVersion(
```

Comment:
It seems wrong the expireObjectEntryVersion exposed inside _objectEntryService. Should it have this name? Or should we call the one from objectEntryVersionService?